app/src/main/java/com/example/mylittlelibrary/data/Book.kt

```diff
@@ -2,4 +2,4 @@ package com.example.mylittlelibrary.data
 
 import java.util.*
 
-data class Book(var name : String, var lendTo: String, var date: Date)
+data class Book(var Id : Int, var name : String, var ISBN : String)
```

Comment:
Changing the constructor signature in a code review is risky.  What's the reason?

Reply:
This will make leveraging available Restful Book  API such as:
https://www.googleapis.com/books/v1/volumes?q=Hamlet easier. Maintaining a separate dto to store borrower (user) information with a foreign key can better help data organization and establish primary-foreign key relationship.